lib/src/main/java/uk/gov/di/ipv/cri/passport/library/helpers/LogHelper.java

```diff
@@ -46,6 +48,14 @@ public static void attachPassportSessionIdToLogs(String sessionId) {
         attachFieldToLogs(LogField.PASSPORT_SESSION_ID_LOG_FIELD, sessionId);
     }
 
+    public static void attachGovukSigninJourneyIdToLogs(String govukSigninJourneyId) {
+        if (StringUtils.isNullOrEmpty(govukSigninJourneyId)) {
+            attachFieldToLogs(LogField.GOVUK_SIGNIN_JOURNEY_ID, "unknown");
```

Comment:
Might have missed it but not seeing any code that sets this default value in the audit events? (assuming we want it to be there too)

Reply:
Yeah good shout I have missed that. I'll add something in the constructor of the AuditEvent class I think to initialise it to a default value when missing